src/SQLQueryStress/Form1.cs

```diff
@@ -354,17 +355,16 @@ private void OpenConfigFile(string fileName)
             }
             catch (Exception exc)
             {
-                MessageBox.Show(string.Format("{0}: {1}", Resources.ErrLoadingSettings, exc.Message));
+                MessageBox.Show($"{Resources.ErrLoadingSettings}: {exc.Message}");
             }
 
-            var sqlControl = elementHost1.Child as SqlControl;
-            if (sqlControl != null)
+            if (elementHost1.Child is SqlControl sqlControl)
             {
                 sqlControl.Text = _settings.MainQuery;
             }
             threads_numericUpDown.Value = _settings.NumThreads;
             iterations_numericUpDown.Value = _settings.NumIterations;
-            queryDelay_textBox.Text = _settings.DelayBetweenQueries.ToString();
+            queryDelay_textBox.Text = _settings.DelayBetweenQueries.ToString(CultureInfo.CurrentCulture);

```

Comment:
Use Invariant - for example to avoid https://stackoverflow.com/questions/2720024/int-parse-of-8-fails-int-parse-always-requires-cultureinfo-invariantculture